tests/foreman/cli/test_host.py

```diff
@@ -2207,7 +2207,7 @@ def setUpClass(cls):
             u'lifecycle-environment-id': cls.env['id'],
             u'activationkey-id': cls.activation_key['id'],
             u'subscription': cls.subscription_name,
-        })
+        }, force_use_cdn=True)
```

Comment:
Pls explain why is this necessary. We'll better avoid using cdn version where it's possible as we have to test the latest changes which will be delivered altogether with satellite to avoid regressions.